src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -79,17 +79,28 @@ object EMD extends DebugEnhancedLogging {
   }
 
   private def addPrivacySensitiveRemark(emd: EasyMetadata, agreementsXml: Elem): Unit = {
-    val signerId = agreementsXml \ "depositAgreement" \ "signerId"
-    val easyAccount = (signerId \@ "easy-account").toOption
-    val fullname = signerId.text
+
+    def userNamePart: String = {
+      val signerId = agreementsXml \ "depositAgreement" \ "signerId"
+      (signerId.text, (signerId \@ "easy-account").toOption, (signerId \@ "email").toOption) match {
+        case (fullName, None, None) => fullName
+        case ("", None, Some(email)) => email
+        case ("", Some(account), None) => account
+        case ("", Some(account), Some(email)) => s"$account ($email)"
+        case (fullName, None, Some(email)) => s"$fullName ($email)"
+        case (fullName, Some(account), None) => s"$account ($fullName)"
```

Comment:
in the context of the text, I would suggest

```suggestion
        case (fullName, Some(account), None) => s"$fullName (username $account)"
```